module-2/backend/week_one.md

```diff
@@ -5,30 +5,109 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+
+* GET - A basic request to retrieve a page from the server
+* PUT - Client is sending information to the server
+* POST - 

```

Comment:
POST - sends information to the database. PUT - updates information 

also, thinking about these as retrieving data from a page rather than just a page, helps in how we imagine these relationships